content/en/events/upcoming-events/gsoc-proposal-template.md

```diff
@@ -0,0 +1,66 @@
+# Project Title
+
+[Provide a exact same title for your project from the proposals page; including the project number as is]
+
+## Summary
+
+[Offer a brief summary of your project, highlighting the problem it addresses, your proposed solution, and the anticipated impact.]
+
+## Personal Information
+
+- **Full Name**: [Your Full Name]
+- **Email Address**: [Your Email Address]
+- **GitHub Profile**: [Your GitHub Profile URL]
+- **LinkedIn**: [Your LinkedIn Profile URL]
+- **University/College**: [Your University/College Name]
+- **Degree Program**: [Your Degree Program]
+- **Year of Study**: [Your Current Year of Study]
+- **Country of Residence**: [Your Country]
+
+## Qualifications; Why Me?
+
+[Highlight your relevant skills, experiences, and qualifications that make you a suitable candidate for this project. Mention any previous work, open-source contributions, or academic achievements that showcase your capabilities. List any/all contributions to Kubeflow community so far.]
+
```

Comment:
I'd love to have a section about motivations to learn more about why they are interested in working on the project.